queries/rst/highlights.scm

```diff
@@ -58,8 +65,30 @@
   ; https://docutils.sourceforge.io/docs/ref/rst/roles.html
   "^:(emphasis|literal|code|math|pep-reference|rfc-reference|strong|subscript|superscript|title-reference|raw):$"))
 
+; Prefix role
+((interpreted_text
+  (role) @_role
+  "interpreted_text" @text.emphasis)
+ (#eq? @_role ":emphasis:"))
+
+((interpreted_text
+  (role) @_role
+  "interpreted_text" @text.strong)
+ (#eq? @_role ":strong:"))
+
+; Sufix role
+((interpreted_text
+  "interpreted_text" @text.emphasis
+  (role) @_role)
```

Comment:
Not sure if there is a convention for this, capturing a node, but not using it for the query itself.

Reply:
why not leave `@_role` away then?